src/commands/provar/runtests.ts

```diff
@@ -0,0 +1,167 @@
+
+// Load the specific messages for this file.
+const messages = Messages.loadMessages('@provartesting/provardx', 'runtests');
+export default class RunTests extends SfdxCommand {
+    public static description = messages.getMessage('commandDescription');
+    public static examples = [
+        "$ sfdx provar:runtests -f './myproperties.json'"
+    ];
+
+    public static args = [{ name: 'file' }];
+
+    protected static flagsConfig = {
+        // flag with a value (-f, --filespec=VALUE)
+        filespec: flags.string({
+            char: 'f',
+            description: messages.getMessage('fileSpecFlagDescription')
+        }),
+        // flag with a value (-p, --propertyfile=VALUE)
+        propertyfile: flags.string({
+            char: 'p',
+            description: messages.getMessage('propertyFileFlagDescription')
+        }),
+        // flag with a value (-o, --connectionoverridefile=VALUE)
+        connectionoverridefile: flags.string({
+            char: 'o',
+            description: messages.getMessage('connectionOverridefile')
+        }),
+        // flag with a value (-c, --cachepath=VALUE)
+        cachepath: flags.string({
+            char: 'c',
+            description: messages.getMessage('cachePathFlagDescription')
+        }),
+        // flag with a value (-m, --metadatalevel=VALUE)
+        metadatalevel: flags.string({
+            char: 'm',
+            description: messages.getMessage('metadataLevelFlagDescription')
+        }),
+        // flag with a value (-s, --secrets=VALUE)
+        secrets: flags.string({
+            char: 's',
+            description: messages.getMessage('secretsFlagDescription')
+        }),
+        // flag with a value (-l, --loglevel VALUE)
+        loglevel: flags.string({
+            char: 'l',
+            description: messages.getMessage('loglevelFlagDescription')
+        })
+    };
+
+    public async run(): Promise<AnyJson> {
+        // const fileSpec : string = this.flags.filespec;
+        const propertyFile: string = this.flags.propertyfile;
+        // const connectionOverrideFile : string = this.flags.connectionoverridefile;
+        const cachePath: string = this.flags.cachepath;
+        const metadataLevel: string = this.flags.metadatalevel;
+        // const secrets : string = this.flags.secrets;
+        // const logLevel : string = this.flags.loglevel;
+        // const json : string = this.flags.json;
+
+        const provarDxUtils: ProvarDXUtility = new ProvarDXUtility();
+        const isValid: boolean = provarDxUtils.validatePropertiesJson(
+            propertyFile
+        );
+
+        if (
+            provarDxUtils.getProperties().testPlan &&
+            provarDxUtils.getProperties().connectionOverride
+        ) {
+            const selection = await cli.prompt(
+                'Test plans detected, connection overrides will be ignored, do you wish to continue (Y/N)? '
+            );
+            if (selection === 'n' || selection === 'n') {
```

Comment:
What's the between of either side of the or operator?

Reply:
Done